ci/docker/os-image-stemcell-builder-jammy/Dockerfile

```diff
@@ -57,14 +57,17 @@ RUN groupadd -o -g ${GROUP_ID} ubuntu                  \
   && echo 'ubuntu ALL=NOPASSWD:ALL' >> /etc/sudoers
 
 # we install a local ruby (3.1) because Jammy is lagging at 2.7
```

Comment:
This comment is stale; you can change to `we install a local Ruby (3.x) because Jammy is lagging at 2.7`.

Note: I probably wrote this comment originally, and I take full responsibility.